usr/sbin/pkcscca/pkcscca.c

```diff
@@ -1537,11 +1538,15 @@ int find_wrapped_keys(CK_FUNCTION_LIST *funcs, CK_SESSION_HANDLE sess,
         if (ulObjectCount == 0)
             break;
 
-        handles = realloc(handles, sizeof(CK_OBJECT_HANDLE) * (++ulTotalCount));
```

Comment:
The above free(handles) needs a NULL check.

Reply:
Good catch, fixed.